compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -376,12 +390,13 @@ generateS390CompareAndBranchInstruction(TR::CodeGenerator * cg,
  * You can force the TR::InstOpCode::C* + TR::InstOpCode::BRC to be generated by setting the needsCC
  * parameter to true (the default).
  */
+template <typename constInt>
```

Comment:
`constInt` is representing the type of immediate value operand for the given Compare and Branch instruction. Can we use more readable name here instead of `constInt`